docs/pages/docs/guides/beginners-guide.mdx

```diff
@@ -0,0 +1,89 @@
+
+## Installing packages
+
+A common issue beginners are facing is the installing of packages in a specific workspace.
+When not working in a monorepo you can simply run `npm install <package>` but now that installs that in the root.
+No when you go in the folder of a workspace and run that command the entire `node_modules` folder will also end up there.
+So how do you install a package?
+
+Well you run the following commands for the following package managers:
+
+| Package manager | Command                                    | Example                        |
+| --------------- | ------------------------------------------ | ------------------------------ |
+| npm             | `npm install <package> -w=<workspace>`     | `npm install react -w=web`     |
+| yarn            | `yarn workspace <workspace> add <package>` | `yarn workspace web add react` |
+| pnpm            | `pnpm add <package> --filter <workspace>`  | `pnpm add react --filter web`  |
+
+## Adding workspaces
+
+Another common issue is that users don't know how to add a workspace to the monorepo.
+
+To add a workspace you need a few things.
+
+1. A folder for your workspace
+2. Something that indicates that a folder is a workspace
+3. A `package.json` file in the workspace folder
+
+### A folder for your workspace
+
+First you need to decide where you want to put your workspace.
+A common pattern is to have an `apps` folder and a `packages` folder.
+The `apps` folder will contain workspaces that house launcheable apps, such as a next.js app.
+The `packages` folder will contain workspaces that house packages that are used by either an app or another package.
+
+When you have decided where to put the workspace you simply create a folder for it.
+
+### Something that indicates that a folder is a workspace
+
+After creating your folder you need to make sure your package manager knows that your folder is a workspace folder.
+
+This step is different for `pnpm` than it is for `npm` and `yarn`.
+
+For `pnpm` you need to add the folder to the `pnpm-workspace.yaml` file that exists in your root.
+This file contains a list of workspaces in the form of globs.
+
+For `yarn` and `npm` you need to add the folder to the `workspaces` field in the root `package.json` file
+This field contains a list of workspaces in the form of globs.
+
+Each folder that matches any of the globs in the lists is considered a workspace folder.
+
+### A `package.json` file in the workspace folder
+
+After creating the folder and adding it to the list of workspaces we want to create the `package.json` file for the workspace.
+This `package.json` file will house all the information for the workspace such as the version, dependencies, and scripts, just like a normal `package.json` file would.
+
+## Naming workspaces
+
+Some issues can occur when choosing names for your workspaces.
+Therefor, we want to provide some guidelines and best-practices.
```

Comment:
Therefore

Reply:
I'll solve it tomorrow. And thanks! My spelling check actually seems to be doing this one wrong so now I can fix it in my graduation thesis :D